app/javascript/components/clickable-dropdown/index.vue

```diff
@@ -54,6 +63,14 @@ export default {
       this.$refs.parentDropdown.closeDropdown();
       this.$emit('item-clicked', event);
     },
+    nextOrgIndex(org_index) {
+      for (var i = org_index + 1; i < this.items.length; i++) {
```

Comment:
All 'var' declarations must be at the top of the function scope  vars-on-top<br>Unexpected var, use let or const instead                         no-var<br>Identifier 'org_index' is not in camel case                      camelcase